### ISSUE TITLE
Problem with timestamp

### ISSUE THREAD

**Issue by kbhCGI** [scandihealth/lpr3-docs#192]
Hi

We get this error.
![image](https://user-images.githubusercontent.com/41666628/46195776-7fbac700-c305-11e8-8720-886d4c722b1d.png)
Is the problem that we have a timestamp in the message, that is after the timestampe on your machine.
ex. my machine 10:00:00, you machine 09:59:58

**Comment by TueCN** [comment]
This is rule 00.01 in [Logisk datamodel og indberetningsregler - Bilag 1 (PDF)](https://sundhedsdatastyrelsen.dk/-/media/sds/filer/rammer-og-retningslinjer/patientregistrering/indberetning-til-lpr3/vejledning-lpr3-bilag-1-model-og-indberetningsregler.pdf?la=da)

The problem it seems is that the `Slutdato` value of the patient's bopæl is in the future. 

**Note:** Rule 00.01 also validates that all timestamps lie before `indberetningens tidsstempel` which is defined as:
> En given LPR3-indberetnings tidsstempel skal oplyse, hvornår selve indberetningen er genereret af
> fødesystemet. Dette er ikke det samme tidspunkt som selve indsendelsen/transmissionen af
> indberetningen, der altid - også i et online-indberetningsmiljø - vil ligge efter tidspunktet for generering

`indberetningens tidsstempel` is thus the effectiveTime value of the CDA [ClinicalDocument](http://lpr-art-decor.westeurope.cloudapp.azure.com:8080/art-decor/decor-templates--lpr-?section=templates&id=1.2.208.176.7.1.10.71&effectiveDate=2018-01-04T00:00:00) element. 
This value is not currently visible on the UI (it is used in validation and stored in the database).

This means you must make sure the ClinicalDocument's effectiveTime timestamp is chronologically after all other timestamps in the submission.

**Comment by TueCN** [comment]
**RESOLUTION**
Currently both startdato (hl7:low) and slutdato (hl7:high) are mandatory when a hl7:usablePeriod element is provided on the hl7:addr element. 
We have just discussed this internally and decided to make both timestamps optional. The [art-decor template](http://lpr-art-decor.westeurope.cloudapp.azure.com:8080/art-decor/decor-templates--lpr-?section=templates&id=1.2.208.176.7.1.10.71) has been updated and this change will go into effect with the next deploy.
